src/app/settings/keyboard-mapping-settings/components/edit-keyboard-mapping/edit-keyboard-mapping.component.html

```diff
@@ -1,12 +1,47 @@
 <sofie-row class="mb-3">
   <form [formGroup]="actionTriggerForm" class="left-container">
     <div [formGroupName]="'data'">
-      <sofie-text-input i18n-label label="global.label" [formControlName]="'label'"></sofie-text-input>
-      <sofie-keyboard-input [isRequired]="true" i18n-label label="action-triggers.shortcut.label" [formControlName]="'keys'" i18n-errorMessage errorMessage="action-triggers.shortcut.error">
+      <sofie-text-input
+        i18n-label
+        label="global.label"
+        [formControlName]="'label'">
+      </sofie-text-input>
+      <sofie-keyboard-input
+        [isRequired]="true"
+        i18n-label
+        label="action-triggers.shortcut.label"
+        [formControlName]="'keys'"
+        i18n-errorMessage
+        errorMessage="action-triggers.shortcut.error">
       </sofie-keyboard-input>
-      <sofie-keyboard-input i18n-label label="action-triggers.physical-mapping.label" [formControlName]="'mappedToKeys'" i18n-helpText helpText="action-triggers.physical-mapping-help.tooltip">
+      <sofie-button-group>
```

Comment:
You only have a single button, so you shouldn't be needing a button-group

Reply:
(Looking through the codebase we might need some cleanup if we are to be in compliance with this rule)